src/components/challenge-rewards-drawer/ChallengeRewardsDrawer.tsx

```diff
@@ -206,7 +206,7 @@ type ChallengeRewardsDrawerProps = {
   /** The number of steps the user has to complete in total */
   stepCount?: number
   /** The amount of $AUDIO that is rewarded to the user for completing the challenge */
-  amount: string
+  amount: number
```

Comment:
Why did this change?

Reply:
yeah i was also curious about that, meant to request marcus' review, cause i think that is around code he is familiar with

Reply:
Yep, new client change went in that changes this to a number: https://github.com/AudiusProject/audius-client/pull/808

Reply:
Thanks for changing this for me 😄

Reply:
perfect!